Cargo.toml

```diff
@@ -90,9 +96,24 @@ members = [
 	"node/test/polkadot-simnet/common",
 	"node/test/polkadot-simnet/node",
 	"node/test/polkadot-simnet/test",
+	"rpc",
+	"parachain",
 	"parachain/test-parachains",
 	"parachain/test-parachains/adder",
 	"parachain/test-parachains/adder/collator",
+	"parachain/test-parachains/halt",
+	"bridges/bin/runtime-common",
+	"bridges/modules/dispatch",
+	"bridges/modules/grandpa",
+	"bridges/modules/messages",
+	"bridges/primitives/chain-rococo",
+	"bridges/primitives/chain-wococo",
+	"bridges/primitives/header-chain",
+	"bridges/primitives/message-dispatch",
+	"bridges/primitives/messages",
+	"bridges/primitives/polkadot-core",
+	"bridges/primitives/runtime",
+	"bridges/primitives/test-utils",
```

Comment:
I'm not sure about bridges pallets as it's added as a subtree as was removed here last time by @HCastano 
cc @svyatonik @acatangiu

Reply:
Yeah I already discussed this with @FlorianFranzen, but he still doesn't has removed them..

Reply:
Imo there's no harm in adding these crates to workspace - they're built with the code anyways. We should avoid adding all `bridges/*` crates - there are crates that shall not be built here (ideally we should remove them from the polkadot repo, but ...).